etc/cmake/options.cmake

```diff
@@ -322,6 +322,16 @@ if(OT_UDP_FORWARD)
     target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_UDP_FORWARD_ENABLE=1")
 endif()
 
+option(OT_PACKAGE_NAME "enable vendor package name")
```

Comment:
@hemanth-silabs Please merge main again into your branch.  This piece of code should be removed.